.github/ci_settings.py

```diff
@@ -14,19 +14,19 @@
 # move to a new minor version below
```

Comment:
Could you enable the upload, as mentioned in the comment? It should work now with the new PostgreSQL version.

Reply:
Fixed